README.md

```diff
@@ -22,45 +21,60 @@ FLAGS:
-    epoch     Generate epoch data for ethash
+    epoch     Generate epoch data for ethereum ethash consensus
     export    Exports shadow's rocksdb
     help      Prints this message or the help of the given subcommand(s)
     import    Imports mmr from shadow backup or geth
     run       Start shadow service
     trim      Trim mmr from target leaf
 ```
 
-
 ## Download
 
-```sh
+``` sh
 $ cargo install darwinia-shadow
 ```
 
-
 ### Note
 
-+ Please make sure you have `golang` installed in your machine
-
+* Please make sure you have `golang` installed in your machine
 
 ## Environment Variables
 
-- `ETHEREUM_RPC`
+> Please use different environment variables for each shadow instance that connects to various blockchain
+
+* `ETHEREUM_RPC`
 
-    Optional. The RPC endpoint of a etherum node, only `http://` and `https://` are supported. Default is http://localhost:8545 .
+    Required only when start shadow service for etherum mmr generation. The RPC endpoint of a etherum node, only `http://` and `https://` are supported. Default is http://localhost:8545.
 
     Example: `http://localhost:8545/`
 
-- `MMR_LOG`
+* `HECO_MAINNET`
 
-    Optional. Define how frequently it outputs logs `Pushed mmr ... into database` while generating MMR. Useful when you first time running shadow, since it generates millon of MMR data at first launch. Default is `10000`.
+    Required only when start shadow service for [heco](https://github.com/HuobiGroup/huobi-eco-chain) mainnet mmr generation. The RPC endpoint of a heco mainnet node, only `http://` and `https://` are supported. Default is https://http-mainnet-node.huobichain.com.
 
-    Example: `"100000"`
+* `HECO_TESTNET`
+
+    Required only when start shadow service for [heco](https://github.com/HuobiGroup/huobi-eco-chain) testnet mmr generation. The RPC endpoint of a heco testnet node, only `http://` and `https://` are supported. Default is https://http-testnet.huobichain.com.
+
+* `BSC_MAINNET`
+
+    Required only when start shadow service for [bsc](https://github.com/binance-chain/bsc) mainnet mmr generation. The RPC endpoint of a bsc mainnet node, only `http://` and `https://` are supported. Default is https://bsc-dataseed.binance.org.
 
+* `BSC_TESTNET`
+
+    Required only when start shadow service for [bsc](https://github.com/binance-chain/bsc) testnet mmr generation. The RPC endpoint of a bsc testnet node, only `http://` and `https://` are supported. Default is https://data-seed-prebsc-1-s1.binance.org:8545.
+
+* `MMR_LOG`
+
+    Optional. Define how frequently it outputs logs `Pushed mmr ... into database` while generating MMR. Useful when you first time running shadow, since it generates millon of MMR data at first launch. Default is `10000` .
+
+    Example: `"100000"`
 
 ## Trouble Shooting
 
-Everytime you run `proof` in error, please delete `~/.ethashproof` and `~/.ethash` 
+Everytime you run `proof` in error, please delete `~/.ethashproof` and `~/.ethash`
```

Comment:
these two files have been moved to `~/.shadow/`

Reply:
ok